core/object/script_language.h

```diff
@@ -312,20 +312,29 @@ class ScriptLanguage : public Object {
 	};
 
 	struct CodeCompletionOption {
+		enum Location {
```

Comment:
I would probably be put as an enum of ScriptLanguage and bind it in ScriptLanguageExtension like the other enums, so the extensions can have access to it. Otherwise you dont know what they are from it.